### PR TITLE
feat(zero-downtime): renames title column to name

### PR DESCRIPTION
**Pull request by forbaokhanh**
WHAT: renames the column title in the Movies table to name

WHY: because myagi said so.

### CODE REVIEW

File: db/migrations/20170525160754_add_name_to_movies.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+};
+
+exports.down = function (Knex, Promise) {
+  return Knex.schema.table('movies', (table) => {
+    table.dropColumn('name');
+  })
+  .then(() => {
+    return Knex.raw('ALTER TABLE movies ALTER COLUMN title SET NOT NULL');
+  });
+};
```
Comment (itsachen) [inline db/migrations/20170525160754_add_name_to_movies.js]: Nit: Add a newline!

File: db/migrations/20170525161804_backfill_movies_name.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+exports.up = function (Knex, Promise) {
+  return Knex.raw('UPDATE movies SET name = title');
+};
+
+exports.down = function (Knex, Promise) {
+  return Promise.resolve();
+};
```
Comment (itsachen) [inline db/migrations/20170525161804_backfill_movies_name.js]: Nit: Add a newline

File: db/migrations/20170525161839_drop_title_from_movies.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+};
+
+exports.down = function (Knex, Promise) {
+  return Knex.schema.table('movies', (table) => {
+    table.text('title');
+  })
+  .then(() => {
+    return Knex.raw('ALTER TABLE movies ALTER COLUMN name DROP NOT NULL');
+  });
+};
```
Comment (itsachen) [inline db/migrations/20170525161839_drop_title_from_movies.js]: Nit: Add a newline

**Review by itsachen — APPROVED** [review]
LGTM
lgtm

**Review by sjl2 — APPROVED** [review]
LGTM

**Pull request merged**